src/shotgunEventDaemon.py

```diff
@@ -1365,6 +1337,12 @@ def _cleanup(self):
 def main():
     """
     """
+    if CURRENT_PYTHON_VERSION <= PYTHON_26:
+        print(
+            "Python 2.5 and older is not supported anymore. Please use Python 2.6 or newer."
+        )
+        return 3
```

Comment:
How come we are using three here? Is it to differentiate from the two we return at the bottom of the method?

Reply:
Yup. The `if __name__` section uses the return value of the `main()` function as the exit code of the program so having distinct non-0 numbers for errors makes things easier to troubleshoot. Old habit.